recipes-debian/connman/connman-gnome_debian.bb

```diff
@@ -0,0 +1,33 @@
+# base recipe: meta/recipes-connectivity/connman/connman-gnome_0.7.bb
+# base branch: warrior
+# base commit: fdbf32bb766c7e7587782f40df79a1ecbd0fed40
+
+SUMMARY = "GTK+ frontend for the ConnMan network connection manager"
+HOMEPAGE = "http://connman.net/"
+SECTION = "libs/network"
+LICENSE = "GPLv2 & LGPLv2.1"
+LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
+
+DEPENDS = "gtk+3 dbus-glib dbus-glib-native intltool-native gettext-native"
+
+BPN = "connman-ui"
+
+inherit debian-package
+require recipes-debian/sources/${BPN}.inc
```

Comment:
Do you need to commit connma-ui.inc file?

Reply:
Yes, I do. I'm sorry I forgot to include the file within the commit. I submit new pull request.